scripts/README.md

```diff
@@ -0,0 +1,51 @@
+# Scripts
+
+## Benchmarking on sift1M
+
+### Download dataset
+
+```shell
+wget http://ann-benchmarks.com/sift-128-euclidean.hdf5
+```
+
+### Run storage backends
+
+```shell
+docker-compose up -d
+```
+
+### Run benchmarking script
+
+Install dependencies:
+
+```
+pip install "docarray[common,benchmark,qdrant,annlite,weaviate,elasticsearch,redis]"
+```
+
+Run the benchmark:
+
+```
+python benchmarking_sift1m.py [-h] [--fixed-hnsw] [--exclude-backends EXCLUDE_BACKENDS]
+
+options:
+  -h, --help            show this help message and exit
+  --fixed-hnsw          Whether to use default HNSW configurations for random dataset
+  --exclude-backends EXCLUDE_BACKENDS
+                        List of comma-separated backends to exclude from the benchmarks
```

Comment:
how do we select sift or random dataset ?

Reply:
I added docs for random dataset now. But I think the scripts for random dataset should be removed in the future.